packages/dev-env/src/seed/quotes.ts

```diff
@@ -0,0 +1,50 @@
+import { SeedClient } from './client'
+import { basicSeed } from './index'
+
+export default async (sc: SeedClient) => {
+  await basicSeed(sc)
+  await sc.createAccount('eve', {
+    email: 'eve@test.com',
+    handle: 'eve.test',
+    password: 'eve-pass',
+  })
+
+  await sc.post(
+    sc.dids.eve,
+    'qUoTe 1',
+    undefined,
+    undefined,
+    sc.posts[sc.dids.alice][0].ref,
+  )
+  await sc.post(
+    sc.dids.eve,
+    'qUoTe 2',
+    undefined,
+    undefined,
+    sc.posts[sc.dids.alice][0].ref,
+  )
+
+  await sc.post(
+    sc.dids.eve,
+    'qUoTe 3',
+    undefined,
+    undefined,
+    sc.replies[sc.dids.bob][0].ref,
+  )
+
+  const spamPosts: Promise<any>[] = []
+  for (let i = 0; i < 50; i++) {
+    spamPosts.push(
+      sc.post(
+        sc.dids.eve,
+        `MASSIVE QUOTE SPAM ${i + 1}`,
+        undefined,
+        undefined,
+        sc.posts[sc.dids.alice][1].ref,
+      ),
+    )
+  }
+  await Promise.all(spamPosts)
```

Comment:
Is the "quote spam" just used to test pagination?  If so you may be able to use smaller pages, as we do some other places.  One upside is that it makes the test suites more reliable and quicker.  Doing all the writes in parallel also _could_ cause one of them to fail, as that is how the PDS bails if it struggles to serialize the writes to a given repo.

Reply:
ha yea didnt realize the writes were made one by one till yesterday 😅